File: C/brocode/user_input.c
#include <stdio.h>

int main() {
    int age;
    char name[24];

    printf("What's your name?\n");
    // scanf("%s", name); //scanf doesn't read whitespaces
    
    fgets(name, 24, stdin);
    printf("Hello %s\n", name);

    printf("How old are you?: \n");
    scanf("%d", &age);

    printf("You are %d years old\n", age);

    return 0;
}

File: Dart/exercises/user_input.dart
import 'dart:io';

void main() {
  print("Enter Your name:");
  var name = stdin.readLineSync();
  print("Enter Your age");
  int age = int.parse(stdin.readLineSync()!);
  age = 100 - age;
  print("$name, You have $age years to be 100 years old");

}